src/applyLookupTable.cc

```diff
@@ -38,10 +38,10 @@ int applyLookupTable(
     PixelT indOffset
 ) {
     int numOutOfRange = 0;
-    auto const maxLookupCol = table.size() - 1;
+    int const maxLookupCol = table.size() - 1;
```

Comment:
Hmm... will this work if `table` is empty? You end up with `table[-1]` down below...